code/game/objects/items/weapons/material/swords.dm

```diff
@@ -15,28 +15,60 @@
 	hitsound = 'sound/weapons/bladeslice.ogg'
 	base_parry_chance = 50
 	melee_accuracy_bonus = 10
+	var/draw_handle
+
+/obj/item/material/sword/update_force()
+	if(material?.hardness < MAT_VALUE_HARD)
+		edge = 0
+		attack_verb = list("attacked", "stabbed", "jabbed", "smacked", "prodded")
+		hitsound = 'sound/weapons/pierce.ogg'
+	if(material?.hardness < MAT_VALUE_RIGID)
+		sharp = 0
+		attack_verb = list("attacked", "smashed", "jabbed", "smacked", "prodded", "bonked")
+		hitsound = "chop"
+	. = ..()
+	
+/obj/item/material/sword/update_icon()
+	. = ..()
+	if(applies_material_colour)
+		if(draw_handle)
+			add_overlay(get_mutable_overlay(icon, "[icon_state]_handle"))
+		if(material.reflectiveness >= MAT_VALUE_SHINY)
+			add_overlay(get_mutable_overlay(icon, "[icon_state]_shine"), adjust_brightness(color, 20 + material.reflectiveness))
+
+/obj/item/material/sword/experimental_mob_overlay(mob/user_mob, slot)
+	var/image/res = ..()
+	//Do not color scabbarded blades
+	if(applies_material_colour && (slot == slot_back_str || slot == slot_belt_str))
+		res.color = null
+	return res
+
+/obj/item/material/sword/wood
+	material = MAT_WOOD
+	draw_handle = FALSE
```

Comment:
Do you want to also make player-made woodcrafts set this?

Reply:
currently sords arent' craftable, so will think of some way to set handles when doing that.